src/pages/guides/getting_started/code_playground.md

```diff
@@ -27,83 +27,64 @@ contributors:
+- **Learners**: Those who are learning the Document APIs and want to see immediate results of their code
+- **Experienced Developers**: Seasoned developers who want to test specific API functionality or debug isolated code snippets
+- **Designers**: UX/UI designers who want to experiment with add-on interfaces without extensive coding setup
 
-The Code Playground is designed to support the following development workflow use cases:
+## Getting Started
 
-- **Experiment First**: Test your ideas and API interactions before committing to full add-on development.
-- **Learn as You Go**: Master the basics of the Document APIs and [Add-on SDK](../../references/addonsdk/index.md) without complex setup requirements.
-- **Prototype Quickly**: Build and test features in minutes instead of hours with instant feedback.
-- **Bridge to Production**: Develop core functionality in Playground before moving to a complete project environment.
-- **Debug with Ease**: Isolate and fix specific issues by testing API calls outside your production code.
+### Prerequisites
 
-## How to Access Code Playground
+Before using Code Playground, ensure you have:
 
-### Step 1: Enable Add-on Development Mode
+- An Adobe Express account
+- A document open in Adobe Express (for testing your code)
 
-- Click the avatar icon in the top right corner of Adobe Express, then the gear icon to open the "Settings".
-- Enable **Add-on Development** if it's not already enabled (you might need to click the **Developer Terms of Use** link the first time).
+### How to Access Code Playground
 
-![Enable Add-on Development](./img/playground-enable-dev-mode.gif)
+#### Step 1: Open Code Playground
 
-### Step 2: Open Code Playground
+1. With any document open in Adobe Express, click the **Add-ons** button in the left rail
+2. Select the **Your add-ons** tab
+3. Toggle on **Code Playground** at the bottom of the panel:
 
-- With any document open, click the **Add-ons** button in the left rail.
-- Select the **Your add-ons** tab.
-- Toggle on **Code Playground** at the bottom of the panel:
+   ![Adobe Express Code Playground Toggle](./img/toggle-playground.png)
 
-  ![Adobe Express Code Playground Toggle](./img/toggle-playground.png)
+4. Once enabled, the playground window will open, allowing you to begin coding immediately:
 
-- Once enabled, the playground window will open, allowing you to begin coding immediately:
+   ![Adobe Express Code Playground Open](./img/playground-open.png)
 
-  ![Adobe Express Code Playground Open](./img/playground-open.png)
+#### Step 2: Enable Add-on Development Mode (if needed)
 
-## Choose Your Development Mode
+If this is your first time using Code Playground or any add-on development features, you'll need to enable Add-on Development mode. See the [Enable Add-on Development Mode](./hello-world.md#enable-add-on-development-mode) section in the Hello, World! guide for detailed instructions.
```

Comment:
We have made changes to simplify this flowl. Developers no longer need to specially go to Settings panel to enable dev mode. If they land on playground using the shorcut URL and havent accepted TOU, they will be intercepted with a dialog to read and accept TOU. 
You can perhaps reword this section letting devs know that they need to accept DTOU (with a screenshot of the dialog) before proceeding. But I wouldnt call this out as a 'Step'.